srcpkgs/anydesk/files/anydesk.xpm

```diff
@@ -0,0 +1,116 @@
+/* XPM */
```

Comment:
Can this be added as distfile instead?

Reply:
Maybe... 
I do not have much time rn, I will try a different approach to fixing the eula checksum issue.

Reply:
I ended up using both the generic tarball (for the license) and the rpm (for the files). Not the most elegant solution, but miles ahead of the previous one...